src/main/java/org/kumoricon/registration/print/ReportPrintService.java

```diff
@@ -0,0 +1,132 @@
+package org.kumoricon.registration.print;
```

Comment:
I like this. If you don't want to do it, I'm going to add a "Download" button that uses this create a downloadable PDF to /admin/tillsessions
(If you do want to do it, feel free. :) )

Reply:
Added to the PR